shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -981,6 +984,112 @@ public void setLocalizationPlugin(@Nullable LocalizationPlugin localizationPlugi
 
   // ----- End Localization Support ----
 
+  // ----- Start Dynamic Features Support ----
+
+  /** Sets the dynamic feature manager that is used to download and install split features. */
+  @UiThread
+  public void setDynamicFeatureManager(@Nullable DynamicFeatureManager dynamicFeatureManager) {
+    ensureRunningOnMainThread();
+    this.dynamicFeatureManager = dynamicFeatureManager;
+    if (dynamicFeatureManager != null) {
+      dynamicFeatureManager.setJNI(this);
+    }
+  }
+
+  /**
+   * Called by dart to request that a Dart deferred library corresponding to loadingUnitId be
+   * downloaded (if necessary) and loaded into the dart vm.
+   *
+   * <p>This method delegates the task to DynamicFeatureManager, which handles the download and
+   * loading of the dart library and any assets.
+   *
+   * @param loadingUnitId The loadingUnitId is assigned during compile time by gen_snapshot and is
+   *     automatically retrieved when loadLibrary() is called on a dart deferred library.
+   */
+  @SuppressWarnings("unused")
+  @UiThread
+  public void requestDartDeferredLibrary(int loadingUnitId) {
+    if (dynamicFeatureManager != null) {
+      dynamicFeatureManager.downloadDynamicFeature(loadingUnitId, null);
+    }
```

Comment:
Since this is where the rubber meets the road, let's show a scary message here so it's easier to debug. i.e. you used a split aot feature by calling the dart loadLibrary function but didn't set a manager. Please see integration doc: .... Split AOT will not work unless you complete integration from native side.

Reply:
You didn't address this one

Reply:
I did, the Github UI just didn't register it as outdated since the comment is on a `}` :)